Base/entry_point.sh

```diff
@@ -11,6 +11,10 @@ if ! whoami &> /dev/null; then
   fi
 fi
 
+# create directories in case of mounted emptyDir
+mkdir -p /var/run/supervisor
+mkdir -p /var/log/supervisor
```

Comment:
It is true that this is not strictly necessary and it probably can be removed.  This is a call made under the assumption the the /var/run or /var/log are mounted as an emptyDir. Then the Subdirectories do not exist. This was done because I was not sure if other logfiles are written outside of the var/log/supervisor directory. Now I am pretty sure this is not the case so this could be removed.
For Example in my case I mount "emptyDir" Volumes at /tmp, /var/tmp /var/run and /var/log, and I use the new configuration option from this patch to move the config file to /tmp/selenium.toml
If we remove this mkdir the emptyDir could either be mounted to /var/run/supervisor and  /var/run/supervisor or the mkdir has to be made in the kubernetes deployment descriptor. Both is not really problematic, so I could remove this if you want.

Reply:
Thanks for your reply, @ptriller1und1.

I understand this might be needed in some cases, but those cases are not the general use case, which is why I think it is better if we remove this bit from the PR. After that, I'd be happy to merge it.

Reply:
I changed it in the merge request.